sig-docs/charter.md

```diff
@@ -0,0 +1,54 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs is responsible for Kubernetes documentation.
+
+The responsibility for feature documentation lies with feature developers. SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and coordinates documentation updates during quarterly releases.
+
+We maintain the Kubernetes website's infrastructure and tooling.
+
+We create subprojects as needed to handle specific aspects of Kubernetes documentation.
+
+### In scope
+
+SIG [readme]
+
+#### Code, Binaries and Services
+
+- The kubernetes.io website, infrastructure and tooling
+
+#### Cross-cutting and Externally Facing Processes
+
+- Standards for content: contribution and reviews
+- Coordinating docs contributions for quarterly releases
+
+### Out of scope
```

Comment:
Is there any boundary between CNCF and the Kubernetes project as far as running or paying for the website or its content goes?

Do branding or style guides of any sort fall under the purview of this SIG?  I guess you sorta call this out in cross-cutting but is there any more specificity to it? eg: https://github.com/kubernetes/community/commits/master/icons which seems to have originated from https://github.com/kubernetes/website/issues/747

Does the content of the Kubernetes blog fall under the purview of this SIG? (https://kubernetes.io/blog/)

Reply:
> Is there any boundary between CNCF and the Kubernetes project as far as running or paying for the website or its content goes?

No. CNCF is paying for the website related costs, such as the Netlify hosting. It's the sole source of funding for SIG Docs as far as I know.

Reply:
> Do branding or style guides of any sort fall under the purview of this SIG? I guess you sorta call this out in cross-cutting but is there any more specificity to it? eg: https://github.com/kubernetes/community/commits/master/icons which seems to have originated from kubernetes/website#747

SIG Docs is responsible for the style guide of the documentation but we are not currently responsible for branding or any Kubernetes org-wide standards. We do have influence and/or sometimes people may run their ideas by us, which may be where that issue and icon change came from.

Reply:
> Does the content of the Kubernetes blog fall under the purview of this SIG? (https://kubernetes.io/blog/)

Yes.

Reply:
@spiffxp 
> Do branding or style guides of any sort fall under the purview of this SIG?

I added a note to the Out of Scope section specifying that they don't.